tsconfig.json

```diff
@@ -2,8 +2,8 @@
 // Documentation: https://www.typescriptlang.org/tsconfig
 {
     "compilerOptions": {
-        // "baseUrl": ".",
-        // "outDir": "./dist/tsc",
+        "baseUrl": "base",
+        "outDir": "./dist/tsc",
```

Comment:
Is this the bundle destination? If yes, I think it's going to break the demo. This out dir is hard-coded in index.html.

Reply:
If it works, we can let as you did.